Readme.md

```diff
@@ -10,10 +10,11 @@ In addition, a classifier is trained to score the matched occurrences of the con
 ## Data Requirements
 The construction for the automaton requires a SKOS thesaurus represented as a `rdflib` `Graph`.
 Concepts should be related to labels by `skos:prefLabel` or `skos:altLabel`.
-In addition it is assumed that concepts are organized in a hierarchy that includes sub-thesauri.
-Concepts and sub-thesauri have to be distinguishable by `rdf:type`.
-The training of the predictor requires labeled text.
+Concepts have to be identifiable be by `rdf:type`.
```

Comment:
Vorschlag: Anschluss mit "and have to be identifiable by" (da ist auch ein "be" zuviel)